tests/jvm/src/test/resources/metac-metacp.index.diff

```diff
@@ -0,0 +1,89 @@
+--- metac
```

Comment:
I'm not sure if my suggestion to diff metac.index and metacp.index was such a good idea after all. It's clearly expected that the output be the two will never be the same, neither top-levels or recorded packages. We can keep it for now, but we might want to consider removing it at some point.

Reply:
Is there any level of similarity that can be expected?  Is it worth doing a partial diff?

Reply:
I don't think so, the differences are inherent and expected

* metac does not emit symbols for *.java sources
* metac supports excluding files
* the relative URIs point to X.scala.semanticdb files in metac but X.class.semanticdb in metacp.